src/components/views/about/AboutIndex.tsx

```diff
@@ -223,9 +226,8 @@ const Middle = styled.div`
 	}
 `;
 
-const UpperButton = styled(ButtonLink)`
-	margin: 50px auto 180px auto;
-	width: fit-content;
+const UpperButton = styled(Button)`
+	margin: 50px auto 180px;
```

Comment:
we should use ButtonLink.

Reply:
It's a button, what's the difference? why we should use ButtonLink?

Reply:
It's a link to another page not a button.